### DISCUSSION TITLE
[아이디어] normal_cell, reduction_cell 선택지에 PASS가 있으면 어떨까?

### DISCUSSION THREAD

**Discussion by afnf33** [bcaitech1/p4-opt-5-vibrhanium-#71] [아이디어]
Special Mission 1: 예시 코드의 `tune.py` 파일을 보고 떠올랐는데, 
``` python
# tune.py
...

    m5 = trial.suggest_categorical(
        "m5",
        ["Conv",
        "DWConv",
        "InvertedResidualv2",
        "InvertedResidualv3",
        "Pass"]
        )
...
```
우리의 현재 `optuna_search.py` 코드에서는 무조건 NC + RC + NC + RC + NC 구조를 이루도록 되어 있는데,

모델 경량화 관점에서 NC나 RC의 선택지에 "아무 block도 쌓지 않는다" 라는 선택지를 넣으면 더더욱 경량화된 모델이 나오지 않을까? 하고 생각이 났습니다.

어떻게 생각하시나요?

**Comment by olenmg** [comment]
근데 NC RC 구조 유지는 해야할거같은게 RC RC 는 잘모르지만 NC NC가 나와버리면 채널수가 증폭돼서 MAC이 엄청 커질거같아요ㅜㅜ 제 생각이 맞을지 의견 부탁드려용

**Reply by afnf33** [reply]
지금 확인했다... 죄송..
MAC을 기준으로 optimize하는 거니까 괜찮지 않을까요? 알아서 NC NC는 피할 것 같은데
